wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

```diff
@@ -220,16 +227,17 @@ The following rules are changed to consistent with rules on `MWC 4k 2022` :
 ### Match procedures
 
 1. A referee will create a multiplayer room 15 minutes before the scheduled match time. Players must join the lobby in that period.
-   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `GBC2 Aut: ({TeamRed}) vs ({TeamBlue})`.
+   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `GBC 2022 Autumn: ({TeamRed}) vs ({TeamBlue})`.
    - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
 2. Each captain can ban **one beatmap** from the pool. These beatmaps can not be picked by any team for the entire duration of the match.
 3. Both captains will alternate picking a beatmap from the mappool.
 4. Each captain must use `!roll` once in `#multiplayer`.
    - The winner of the `!roll` could choose to pick first or ban first of the match. The other choose will be used by loser team.
-5. After each pick, the captain may ban one player of the opposing team from playing that beatmap in the match, **except in a tiebreaker**.
+5. After each pick, both captains **could** ban **one** of the opposing team members from playing this beatmap in this match, **except in tiebreaker**. A player can't be banned until he has finished playing a beatmap on this match.
    - Any player can only be banned like this **once** per match.
-   - A player **can't** be banned until he finished playing a game on this match.
-   - Banning players is **optional**.
+   - The team pick beatmap first can't ban players at the 2nd round (opposite's first pick round).
+     - As the former rules, no teams could ban players in 1st round since they haven't finished playing any beatmap.
+   - Team that is picking beatmaps should ban player first.
```

Comment:
does this rule add anything that isn't covered by the other rules? i'm not sure i'm understanding this one

Reply:
This is an emphasis of this rule:   
> Picking beatmaps and banning players should be sent **at a single message**, if a team continues to be slow at banning, referees have the right to **ignore this ban**.

in case some team doesn't say their bans on the same message, this rule is to hint teams not pick this map needn't say player bans until the opposite is sent.

Reply:
ah i see

```suggestion
   - The team that is currently picking a beatmap gets to ban a player first, before the other team.
```

Reply:
emmmm.... seemed weird but will apply, the conversion leaves open, waiting for more suggestions.